src/main/content/_assets/js/instance.js

```diff
@@ -71,13 +71,17 @@ function loadAllInfo(){
     fetchAllInstances()
         .then(setInstanceSelections)
         .then(fetchAnInstance)
-        .then(updateInstanceView);
+        .then(updateInstanceView)
+        .then(fetchCollectionData)
+        .then(setCollectionCard);
```

Comment:
this is outdated, there is a function `handleCollectionsRequests` that is called here where all the "getters" should be called.